daemon/job/snapjob.go

```diff
@@ -0,0 +1,379 @@
+package job
+
+import (
+	"context"
+	"github.com/pkg/errors"
+//	"github.com/problame/go-streamrpc"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/zrepl/zrepl/config"
+	"github.com/zrepl/zrepl/daemon/filters"
+	"github.com/zrepl/zrepl/daemon/job/reset"
+	"github.com/zrepl/zrepl/daemon/job/wakeup"
+	"github.com/zrepl/zrepl/daemon/logging"
+	"github.com/zrepl/zrepl/daemon/pruner"
+	//"github.com/zrepl/zrepl/pruning"
+	"github.com/zrepl/zrepl/daemon/snapper"
+	"github.com/zrepl/zrepl/daemon/transport/connecter"
+	"github.com/zrepl/zrepl/endpoint"
+//	"github.com/zrepl/zrepl/replication"
+	"github.com/zrepl/zrepl/util/envconst"
+	"github.com/zrepl/zrepl/zfs"
+	"sync"
+	"time"
+)
+
+type snap_ActiveSide struct {
```

Comment:
Go naming convention is CamelCase for public structs, camelCase for private structs.
I'd suggest you name this struct `SnapJob`, see common on `snap_activeMode` for why we don't need any active / passive sides for this kind of job.